integration-tests/testutils/build/build_test.go

```diff
@@ -263,3 +264,15 @@ func (s *BuildSuite) TestAssetsDoesNotBuildSnapdFromBranchIfNotInstructedTo(c *c
 		check.Commentf("Expected 0 call to execCommand with %s, got %d",
 			buildSnapdCmd, buildCall))
 }
+
+func (s *BuildSuite) TestAssetsHonoursBuildTags(c *check.C) {
+	testBuildTags := "mybuildtag"
+	Assets(s.useSnappyFromBranch, s.arch, testBuildTags)
```

Comment:
I noticed, but `build.Assets` still reads as a noun to me (_"these are the build assets"_) rather than a verb. Besides, it's being used in contexts without the package name too, which is even less helpful. Code will definitely become more clear if you are explicit about it.